src/tests/test_volunteer_profile.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+
+class TestVolunteerProfile:
+
+    url = reverse("profile")
+
+    @pytest.mark.django_db
```

Comment:
`@pytest.mark.django_db `
можно не ставить перед каждой функцией, достаточно поставить перед всем классом

```suggestion
   @pytest.mark.django_db(transaction=True)
   class TestVolunteerProfile:
```